src/index.html

```diff
@@ -1,13 +1,69 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html class="page"
+  lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible"
+      content="ie=edge">
+    <title>Product card</title>
+    <link rel="stylesheet"
+      href="./styles/index.css">
+  </head>
+  <body class="page__content">
+    <main>
+      <div class="product-container">
+        <section class="product-container__cards">
+          <div
+            class="product-container__card"
+            data-qa="card">
+            <img class="product-container__card-pic"
+              src="./images/imac.jpeg"
+              alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)">
+            <h4 class="product-container__card-title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h4>
+            <p
+              class="product-container__card-product-code">
+              Product code: 195434
+            </p>
+            <div class="stars">
+              <div class="stars__container">
```

Comment:
let's do so, and apply vertical margin to `.card__stars` (to element, not to `stars` which is BEM-block)
```suggestion
            <div class="stars card__stars">
```